src/Core/Services/ApiService.cs

```diff
@@ -528,13 +528,20 @@ private async Task<ErrorResponse> HandleErrorAsync(HttpResponseMessage response,
                 await _logoutCallbackAsync(true);
                 return null;
             }
-            JObject responseJObject = null;
```

Comment:
Discovered by accident when I thought I restarted my DB container but actually stopped it, the `prevalidate` response error comes back as `application/json` so it passes the `IsJsonResponse` test, but the content is not json formatted (it's a SQL backend not found exception) so newton throws.  By returning null it falls back to a generic error instead of crashing the app.